cmake/modules/FindCrossGUID.cmake

```diff
@@ -92,12 +92,12 @@ if(CROSSGUID_FOUND)
     if(CROSSGUID_LIBRARY_RELEASE)
       set_target_properties(CrossGUID::CrossGUID PROPERTIES
                                                  IMPORTED_CONFIGURATIONS RELEASE
-                                                 IMPORTED_LOCATION "${CROSSGUID_LIBRARY_RELEASE}")
+                                                 IMPORTED_LOCATION_RELEASE "${CROSSGUID_LIBRARY_RELEASE}")
```

Comment:
A similar change is missing in FindTinyXML.cmake, FindMySqlClient.cmake and FindMariaDBClient.cmake.

Reply:
Its not required in them as we arent actually using the target created. Its still just using the *_LIBRARIES/*_INCLUDEDIR etc